docs/backup-restore.md

```diff
@@ -205,4 +211,8 @@ If you used `pg_dump` to create the backup file, use the `psql` command to resto
 
 Use the `.restore` command as specified in [Special commands to sqlite3 (dot-commands)](https://www.sqlite.org/cli.html#special_commands_to_sqlite3_dot_commands_).
 </div>
+<div id="YugabyteDB_Managed1" class="tabcontent" markdown="1">
+
+You can restore from the scheduled backup within the backup retention period. See [YugabyteDB Managed: Back up and restore clusters](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-clusters/backup-clusters/).
```

Comment:
YugabyteDB seems to provide a PITR feature.
https://docs.yugabyte.com/preview/manage/backup-restore/point-in-time-recovery/
So, we probably need to mention it here.
(Also, we need to enable the PITR feature by creating a snapshot schedule, so we might need to add a tab for YugabyteDB in the `Back up with explicit pausing` section.) 

But, I'm not sure about the relationships between the usual backup/restore and PITR in YugabyteDB.
It looks like [this doc](https://docs.yugabyte.com/preview/yugabyte-cloud/cloud-clusters/backup-clusters/) doesn't describe PITR and snapshots at all, and the above doc doesn't really describe usual backup and restore.

Could you take a deeper look at the docs and check the features?

Reply:
@feeblefakie Good catch!

I confirmed the PITR feature with local YugabyteDB via `yb-admin` cli command worked fine.

But, as for the managed YugabyteDB, the web console only provides the full and incremental backup/restore not PITR. Also, it seems impossible to conduct the PITR via `yb-admin` cli since the managed YugabyteDB doesn't expose TCP port:7100 which is used for `yb-admin`. I'll ask YugabyteDB support team about it.

Reply:
@feeblefakie I received a reply from Yugabyte team. YugabyteDB Managed doesn't provide PITR feature at the moment. I'll talk with their product team later to get their plan of the feature. I'll update this document once the feature is released in the future.

BTW, I only described YugabyteDB Managed not OSS YugabyteDB since I think it's not easy to deploy the cluster by  users. Do you think we should describe OSS YugabyteDB as well?

Reply:
Thank you!
I understand the current situation.
I think, for now, we can assume users use YugabyteDB Managed.

Reply:
When we back up with explicit pausing, we should probably use on-demand backup during a paused period, 
so I think we should mention so in the backup-side doc.
Here, I think we should probably mention that we restore from the backups taken by the on-demand backups, not from the scheduled backups.
What do you think?

Reply:
@feeblefakie Thanks for the feedbacks! I overlooked the need for explicit pausing when dealing with multiple databases.

I added a description about YugabyteDB with explicit pausing https://github.com/scalar-labs/scalardb/pull/1710/files#diff-0847747898d2c35d3723a35f59e28fbdc704e3d2146f2c5ca5e01850ac410badR136-R141.

Regarding the backup types (on-demand or scheduled backup), I think either is fine as long as the backup timing is synced with explicit pausing. I didn't elaborate it in the newly added description, but let me know if you think we should.